setup.py

```diff
@@ -51,11 +51,15 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        "Plone",
```

Comment:
there might be some hidden functionality coming from Plone (or any of its dependencies 😓 ) as tests do fail 😕

Reply:
Maybe adding Products.CMFPlone, and plone.app.testing should fix this?

Reply:
will try!